gen/builders/state_tracker.go

```diff
@@ -23,37 +30,154 @@ import (
 // messages.
 type StateTracker struct {
 	vector *schema.TestVector
+	bc     *BuilderCommon
+
+	StateTreeVersion types.StateTreeVersion
+	ActorsVersion    actors.Version
 
 	Stores    *Stores
 	StateTree *state.StateTree
 	Driver    *conformance.Driver
 
 	CurrRoot cid.Cid
+
+	EmptyObjectCid       cid.Cid
+	EmptyArrayCid        cid.Cid
+	EmptyMapCid          cid.Cid
+	EmptyMultiMapCid     cid.Cid
+	EmptyBitfieldCid     cid.Cid
+	EmptyDeadlinesCid    cid.Cid
+	EmptyVestingFundsCid cid.Cid
```

Comment:
These CIDs can change based on the actors versions, that's why they are not globals anymore.